src/api/cmd/chart-repo-sync/sync_test.go

```diff
@@ -0,0 +1,447 @@
+package main
+
+import (
+	"archive/tar"
+	"bytes"
+	"compress/gzip"
+	"crypto/rand"
+	"errors"
+	"fmt"
+	"image"
+	"image/color"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"strings"
+	"testing"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/arschles/assert"
+	"github.com/disintegration/imaging"
+	"github.com/kubeapps/common/datastore/mockstore"
+	"github.com/stretchr/testify/mock"
+	"gopkg.in/mgo.v2/bson"
+)
+
+var validRepoIndexYAML = `
+entries:
```

Comment:
Could you store this as another file inside a `testdata` directory?

Reply:
+1 will move it